floss/__init__.py

```diff
@@ -1,6 +1,12 @@
 # Copyright (C) 2017 FireEye, Inc. All Rights Reserved.
 
-from utils import ONE_MB
-from utils import STACK_MEM_NAME
-from utils import makeEmulator
-from utils import removeStackMemory
+import logging
+
+from . import version
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
If you always use `logging.getLogger(__name__)`, the loggers will be kept separate and will propagate to the top-level logger.  This is why I've imported the package `logger` in the CLI app and set a `SteamHandler` to the root `logger` instead of just the `main` module's `logger`.  Having a single root-level package `logger` also allows folks who use `floss` as a module to control FLOSS' logging from one place.

The package-level logger will be named `floss` and the CLI app logger is named `floss.main` because of how `__name__` works.  Log events created in `floss.main` will naturally propagate to ancestor loggers (`floss` in this case).

I'm happy to change this if you want it changed but this is how I've handled logging in very large projects and this is also what the logging library documentation recommends in the [advanced logging tutorial](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial).

I'll look to you for the final call here.